### ISSUE TITLE
Comment with a link to the post in the PR

### ISSUE THREAD

**Issue by targos** [nodejs/bluesky#9]
Like @joyeecheung did manually in https://github.com/nodejs/bluesky/pull/8#issuecomment-2513889888

**Comment by joyeecheung** [comment]
If you mean replying, you can just use the reply action field, see https://github.com/nodejs/bluesky/blob/main/records/new/reply.json.example for an example, and https://github.com/joyeecheung/bluesky-playground/pull/3 to see it in action

**Comment by targos** [comment]
Yeah, I mean comment on GitHub so we can click on the link to see the post on bsky.app

**Comment by joyeecheung** [comment]
The workflow to update to do this is in https://github.com/nodejs/bluesky/blob/main/.github/workflows/process.yml
